lfs/lfs.go

```diff
@@ -29,6 +30,7 @@ var (
 	LocalGitStorageDir string // parent of objects/lfs (may be same as LocalGitDir but may not)
 	LocalMediaDir      string // root of lfs objects
 	LocalObjectTempDir string // where temporarily downloading objects are stored
```

Comment:
I think LocalMediaDir and LocalObjectTempDir should be removed in favour of LocalStorage. Looking through the code I think this identifies a few other places that manual uses of these dirs needs to be abstracted through LocalStorage